linux_os/guide/system/logging/rsyslog_filecreatemode/tests/filecreatemode_0600.pass.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# platform = multi_platform_ubuntu
```

Comment:
I would suggest to remove `platform` restriction in these test scenarios since the tests seem reasonable regardless of product.

Reply:
removed it from all tests